doc/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
windows support Orz....
I think only bash is enough

Reply:
Will remove this file.